pdp/ast/yast/parser_test.go

```diff
@@ -226,10 +226,9 @@ policies:
            - equal:
              - attr: strAttr
              - selector:
-                uri: "local:content/content-item"
+                uri: "pip:mcafee-ts/domain-category"
```

Comment:
In the test check changed from local to pip. We need to check both of them. So please add pip selector but don't remove local selector.

Reply:
Sorry. That was a hasty change and is totally unintentional.